layouts/partials/faq.html

```diff
@@ -0,0 +1,245 @@
+            <div class="single-accordion-item closed">
+              <!-- accordion heading start  -->
+              <div class="accordion-heading">
+                <h3>Can we try KubeDB Enterprise Edition?</h3>
+                <button class="icon">
+                  <i class="fa fa-plus" aria-hidden="true"></i>
+                </button>
+              </div>
+              <!-- accordion heading end -->
+              <!-- accordion body start  -->
+              <div class="accordion-body">
+                <p>Of course! We offer a 30 days license free of cost to try KubeDB Enterprise Edition. And if you
+                  want more trial days to make your decision you can contact us, we are open to talk.</p>
+              </div>
+              <!-- accordion body end -->
+            </div>
+            <!-- single accordion item end  -->
+
+            <!-- single accordion item start  -->
+            <div class="single-accordion-item closed">
+              <!-- accordion heading start  -->
+              <div class="accordion-heading">
+                <h3>Can I upgrade or downgrade my subscription anytime?</h3>
+                <button class="icon">
+                  <i class="fa fa-plus" aria-hidden="true"></i>
+                </button>
+              </div>
+              <!-- accordion heading end -->
+              <!-- accordion body start  -->
+              <div class="accordion-body">
+                <p>Monthly plans are subject to a 30 day minimum term. Monthly plans can be upgraded anytime. Annual
+                  plans can be downgraded at the end of the term.</p>
+              </div>
+              <!-- accordion body end -->
+            </div>
+            <!-- single accordion item end  -->
+
+            <!-- single accordion item start  -->
+            <div class="single-accordion-item closed">
+              <!-- accordion heading start  -->
+              <div class="accordion-heading">
+                <h3>Is my payment information safe?</h3>
+                <button class="icon">
+                  <i class="fa fa-plus" aria-hidden="true"></i>
+                </button>
+              </div>
+              <!-- accordion heading end -->
+              <!-- accordion body start  -->
+              <div class="accordion-body">
+                <p>Our payment processing is powerd by <a href="https://stripe.com/">Stripe</a>. You credit card data
+                  never touches our servers. For
+                  more information, please visit <a href="https://stripe.com/">stripe.com</a>.</p>
+              </div>
+              <!-- accordion body end -->
+            </div>
+            <!-- single accordion item end  -->
+
+            <!-- single accordion item start  -->
+            <div class="single-accordion-item closed">
+              <!-- accordion heading start  -->
+              <div class="accordion-heading">
+                <h3>Can I upgrade or downgrade my subscription anytime?</h3>
```

Comment:
~Can I upgrade or downgrade my subscription anytime?~
Already asked that question!